modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommercePriceLists.testcase

```diff
@@ -1093,14 +1093,138 @@ definition {
 			value1 = "7.45");
 	}
 
-	@description = "This is a test for COMMERCE-6220. When creating a price list only catalogs for which the user has permission should be visible in the drop down"
-	@ignore = "true"
+	@description = "This is a test for COMMERCE-8568. Verify multiple Price Entries can be set on the same Price List for the same SKU"
 	@priority = "3"
-	test AllowMultiplePriceEntriesOnTheSamePriceList {
+	test AllowMultiplePriceEntriesInTheSamePriceList {
+		property osgi.module.configuration.file.names = "com.liferay.commerce.price.list.configuration.CommercePriceEntryConfiguration.config";
+		property osgi.module.configurations = "checkInterval=&quot;1&quot;";
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-8568 AllowMultiplePriceEntriesOnTheSamePriceList Pending Implementation
+		task ("Setup: Uncheck Require strangers to verify their email address") {
+			PortalSettings.gotoConfiguration(
+				configurationCategory = "User Authentication",
+				configurationName = "General",
+				configurationScope = "Virtual Instance Scope");
+
+			PortalSettings.uncheckStrangersVerifyEmailCheckbox();
+		}
```

Comment:
This can be removed in favor of the custom properties mentioned in the comment above.